packages/forms-web-app/src/controllers/common/enter-code.js

```diff
@@ -208,6 +220,15 @@ const postEnterCode = (views) => {
 		// if handling an email confirmation
 		// session will be in browser so can redirect here and consider email confirmed
 		if (tokenValid.action === enterCodeConfig.actions.confirmEmail) {
+			if (isReturningUser) {
+				const userEmail = req.session.appeal.email;
+				const associatedAppeals = await getAppealsByEmail(userEmail);
```

Comment:
Need to check in SQL rather than MongoDB

Reply:
You can use the new endpoint:

`/api/v2/users/{user-id}/appeals`